src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -827,3 +827,15 @@
     text: >
         az acr webhook update -n MyWebhook -r MyRegistry --status disabled
 """
+
+helps['acr check-health'] = """
+type: command
+short-summary: Gets health information on the environment and, optionally, a target registry.
+examples:
+  - name: Gets health state with target registry 'MyRegistry', skipping confirmation for pulling image.
```

Comment:
nit:  capitalization of MyRegistry varies.  Probably should make consistent with other samples.

Reply:
Changed to 'MyRegistry', as it is the standard being used.